tutorials/yes-no-game/full game/yesnogame.edn

```diff
@@ -0,0 +1,251 @@
+; SPDX-License-Identifier: BSD-3-Clause
+; Copyright © 2022 Fragcolor Pte. Ltd.
+
+(def left-x-offset -250)
+(def right-x-offset 250)
+(def total-rounds 10)
+(def max-timer 5)
+
+(defshards initialize []
+  (LoadImage "data/cats/cat01.png") (Push :Name .cat-images)
```

Comment:
Where do those images come from? If there are under a license (hopefully something like CC-BY-SA), include a LICENSE file in the `data` folder.

Reply:
I used the images from this site:
https://www.pexels.com/license/

Is there a format I should follow for the licensing?

Reply:
From their terms it seems ok to use the pictures in a tutorial, but I don't like the fact that it is just a few sentences on a website and not a recognized license such as CreativeCommons.

In the future, I'd prefer that we use materials with clear licensing terms.

Reply:
Noted. I will look around for some that we can use 👍🏻